GUI with recording functionality

raeqi opened the pull request.

Commit:
GUI with recording functionality

raeqi commented:
Pull latest GUI updates

Commit:
Connect to 2018ece496 Firebase Project

KatJHuang commented on EmotionRecogSpeechGUI/app/src/main/java/project/ece496/emotionrecogspeechgui/Record.java:
```diff
@@ -0,0 +1,249 @@
+
+    //private OnFragmentInteractionListener mListener;
+
+    public Record() {
```
Functionality implementation looks pretty good!!! 
Would suggest to remove all the commented code cuz too much of them makes reading hard with huge scrolls Dx

Commit:
Upload to storage functionality added

Commit:
Send Text and Audio

- can send both text and audio file to server
- need to make sure "done receiving" then show "Upload Success on GUI"

Commit:
can receive labels now
